examples/mobile/HomeApp/js/app.js

```diff
@@ -48,6 +41,28 @@
 			.then((response) => response.json())
 			.then((data) => {
 				webClipList = data;
+				// make sure that default active webclip list contains only available elements
+				activeWebClipList.forEach(function (webclip) {
+					if (!webClipList.includes(webclip)) {
+						activeWebClipList.splice(activeWebClipList.indexOf(webclip), 1);
+					}
+				})
+			})
+			.catch(() => {
+				// use default webclip list is sth wrong
+				webClipList =
+					[
```

Comment:
I have a question.
If  web clip is removed from the server, is it also updated in the "webClipList"? So "webClipList" don't have the a removed web clip ?

Reply:
both webClipList and activeWebClip list will not have removed web clip.
So it will not be presented in web app.
But please note that this list is updated only on web app start, so after removing webclip form server web app need to be restarted.